src/main/drivers/accgyro_spi_mpu6500.h

```diff
@@ -25,6 +25,7 @@
 #define MPU6500_RA_ACCEL_CFG                (0x1C)
 #define MPU6500_RA_LPF                      (0x1A)
 #define MPU6500_RA_RATE_DIV                 (0x19)
+#define MPU6500_INT_STATUS                  (0x38)
```

Comment:
Yeah but you can read that through SPI. There is no interrupt handling in MPU6500....my question was also for someone who can test to implement.